python/test/assign0/conftest.py

```diff
@@ -0,0 +1,160 @@
+"""
+conftest.py needed for pytest to detect files.
```

Comment:
Add note saying to ignore this file if you are a student. FWIW, this is currently being ignored as per the `.pylintrc` for the linter target. Maybe go and fix the linter errors here too...